code/src/modules/object_tracker.cpp

```diff
@@ -423,4 +607,29 @@ void ObjectTracker::CalculateTrackingTrajectory(FlightController *fc, Vec3D *cou
     fc->cam->SetTrackingArrow({trackx/TRACK_SPEED_LIMIT_X,
         -tracky/TRACK_SPEED_LIMIT_Y, trackw/TRACK_SPEED_LIMIT_W});
 
+
+
+
+
+
+
+
+
 }
+//using North, East, Down coordinates
+Vec3d ObjectTracker::GroundFromGPS(Coord3D coord){
+    Vec3d retval(
+        DEG2RAD(coord.lat - launch_point.lat) * (1000*RADIUS_OF_EARTH),
+        DEG2RAD(coord.lon - launch_point.lon) * (1000*RADIUS_OF_EARTH*cos(RAD2DEG(launch_point.lat))),
+        (launch_point.alt - coord.alt)
+        );
+    return retval;
+}
+Coord3D ObjectTracker::GPSFromGround(Vec3d coord){
+    Coord3D retval;
+    retval.lat = launch_point.lat + RAD2DEG(coord(0)/(1000*RADIUS_OF_EARTH));
+    retval.lon = launch_point.lon + RAD2DEG(coord(1)/(1000*RADIUS_OF_EARTH*cos(RAD2DEG(launch_point.lat))));
```

Comment:
This function basically duplicates `CoordAddOffset`. 

`cos(RAD2DEG(launch_point.lat))` should be `cos(DEG2RAD(launch_point.lat))`. This also applies to `GroundFromGPS`.